liteflow-labs/starter-kit
# Update useBlockExplorer to be multi chain compatible. Remove need for envs `NEXT_PUBLIC_BLOCKCHAIN_EXPLORER_URL` and `NEXT_PUBLIC_BLOCKCHAIN_EXPLORER_NAME`

### Description

Update the `useBlockExplorer` hook to accept a `chainId` and return the proper network based on this chain ID.

I had to create 2 new buttons `AcceptOffer` and `CancelOffer` as the modal was outside of an iteration, and I couldn't get the chain id to display the proper network.

### How to test

Create an offer, a bid, accept an offer, create an nft... and make sure that all the modals have the link to etherscan
